src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -173,15 +172,22 @@ export default {
         focusedImage() {
             this.trackDuration();
             this.resetAgeCSS();
+        },
+        listenToKeys(val) {
```

Comment:
Why doesn't the mouse over just call a method that focuses on the element? I don't think we need to track any state here? Only one thing can be focused at once, so calling focus on another element elsewhere will automatically blur this one.

Reply:
Got it!